metricflow/query/query_parser.py

```diff
@@ -158,6 +159,18 @@ def parse_and_validate_query(
         finally:
             logger.info(f"Parsing the query took: {time.time() - start_time:.2f}s")
 
+    def _validate_no_dimension_query_for_metric(self, metric_references: Sequence[MetricReference]) -> None:
+        """Validate if all requested metrics are queryable without any dimensions."""
+        for metric_reference in metric_references:
+            metric = self._metric_semantics.get_metric(metric_reference)
+            if metric.type == MetricType.CUMULATIVE:
+                # Cumulative metrics configured with a window/grain_to_date cannot be queried without a dimension.
+                if metric.type_params.window or metric.type_params.grain_to_date:
+                    raise UnableToSatisfyQueryError(
+                        f"Metric {metric.name} is a cumulative metric specified with a window/grain_to_date "
+                        f"which cannot be queried without a dimension.",
```

Comment:
Why not? Is this because they require a time dimension? If there's a specific dimension requirement (e.g., must include `metric_time`) it'd be good to update the error message accordingly.